<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgTrainTable</class>
 <widget class="QDialog" name="DlgTrainTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>列车数据</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>481</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainLayout" stretch="1,4">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,2,1,1,2,1,1,5,0">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>车号：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblTrain">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>001</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>速度：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>300km/h</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>日期：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblDateTime">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>2017-12-24 13:30</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="btnDatasExport">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="text">
         <string>数据导出</string>
        </property>
        <property name="icon">
         <iconset resource="../images.qrc">
          <normaloff>:/images/3Doutput.png</normaloff>:/images/3Doutput.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>4</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>5</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>6</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>7</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>8</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>9</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>10</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>计轴</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>内侧距</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>车轮号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>轮径</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>最大径Dmax</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>最小径Dmin</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>径跳</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>轮缘高Sh</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>轮缘厚Sd</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>轮缘综合值Qr</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>同轴轮径差ΔD1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>同架轮径差ΔD2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>同车轮径差ΔD3</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>计轴</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>内侧距</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="0" column="2">
       <property name="text">
        <string> 车轮号</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="0" column="3">
       <property name="text">
        <string> 轮径D</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="0" column="4">
       <property name="text">
        <string>最大径Dmax</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="0" column="5">
       <property name="text">
        <string>最小经Dmin</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="0" column="6">
       <property name="text">
        <string>径跳ΔR</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="0" column="7">
       <property name="text">
        <string>轮缘高Sh</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="0" column="8">
       <property name="text">
        <string>轮缘厚Sd</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="0" column="9">
       <property name="text">
        <string>轮缘综合值Qr</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="0" column="10">
       <property name="text">
        <string>同轴轮径差ΔD1</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="0" column="11">
       <property name="text">
        <string>同架轮径差ΔD2</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="0" column="12">
       <property name="text">
        <string>同车轮径差ΔD3</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string>（1350-1356）</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="1" column="3">
       <property name="text">
        <string>(770-840)</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </item>
      <item row="2" column="0">
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item row="2" column="2">
       <property name="text">
        <string>d1</string>
       </property>
      </item>
      <item row="3" column="2">
       <property name="text">
        <string>d2</string>
       </property>
      </item>
      <item row="4" column="0">
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item row="4" column="2">
       <property name="text">
        <string>d3</string>
       </property>
      </item>
      <item row="5" column="2">
       <property name="text">
        <string>d4</string>
       </property>
      </item>
      <item row="6" column="0">
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item row="8" column="0">
       <property name="text">
        <string>4</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
